apps/web/src/environment.ts

```diff
@@ -17,6 +17,14 @@ export const Environment = {
     return this.config('chainType', ChainType.TestNet)
   },
 
+  get algoExplorerBaseUrl(): string {
```

Comment:
I think I get `You are not the owner of this NFT.` because I'm running Sandbox i.e. `ALGOD_SERVER=http://localhost` Does it make sense to have a BaseUrl for Sandbox, or the absence of Sandbox intentional?  

![image](https://user-images.githubusercontent.com/3670577/151437518-3046c084-63b7-4029-b416-52afe13766e8.png)

Reply:
This is a tricky one. It's setup to use AlgoExplorer for some queries as I need to rely on the indexer. Because of that, using the sandbox won't work if you wanna try this feature. It _might_ be possible to configure it so that the sandbox mode uses the localhost URLs, but haven't tried it yet.

Reply:
I gave it a whirl, and it appears to work. So I threw together a PR that adds sandbox support with `ALGOD_ENV=sandnet`

https://github.com/deptagency/algomart/pull/253

![image](https://user-images.githubusercontent.com/3670577/151559500-e8d84aed-07c7-4c8d-8e22-54aedae85967.png)